### ISSUE TITLE
Broken app label: AnyDesk

### ISSUE THREAD

**Issue by kai-h** [Installomator/Installomator#1995]
**What's wrong with the app label?**
It looks like the AnyDesk app is now signed with a new Developer ID. The label expects it to be KU6W3B6JMZ whereas the version I have just downloaded from https://download.anydesk.com/anydesk.dmg has ID KHRWM533LU instead.

**Installomator version**
Installomator 10.6 Date 2024-08-30

**Comment by MrSuaz** [comment]
I ran into this same issue. Just go into the script, find the label, update the TeamID to Match. The script will match up and install the app just fine.

**Comment by kai-h** [comment]
Yes, it's a pretty each change - I just wanted to flag this so it gets updated in the next release.
If you're deploying Installomator via an automated method that lets you run a post install script, then this will fix it for you:

`sed -i '' 's/KU6W3B6JMZ/KHRWM533LU/g' /usr/local/Installomator/Installomator.sh`

If you're deploying manually and want to make the change in Terminal, simply prefix it with sudo

`sudo sed -i '' 's/KU6W3B6JMZ/KHRWM533LU/g' /usr/local/Installomator/Installomator.sh`

**Comment by WeDiggITSupportTeam** [comment]
I am experiencing this issue as well. Is there a way to change the Developer ID in the script?

**Comment by kai-h** [comment]
> I am experiencing this issue as well. Is there a way to change the Developer ID in the script?

Yes, if you look at my comment directly above yours there are two ways I’ve given to fix it

**Comment by uurazzle** [comment]
Hi @kai-h:

Thanks for the report.

### Issue fixed on main

Good news -- this has already been fixed on the `main` branch. The label(s) reported in this issue now resolve and download correctly:

- `anydesk` -- version detected: 9.7.0 (installed)

The fix was merged via:
- #2078 -- anydesk.sh: Update Team ID

The updated label will be included in the next Installomator release. In the meantime, you can run from `main` by assembling locally or waiting for the next tagged release. Closing this issue -- feel free to reopen if you continue to see problems. Thanks!
